recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
```

Comment:
Makes sense to me and sounds good. 

FWIW, I have never actually seen a formal release on pypi for this package. Here are the pypi version numbers from the 'simple' interface ([here](https://pypi.python.org/simple/xgboost/)):

```
xgboost-0.4a15.tar.gz
xgboost-0.4a22.tar.gz
xgboost-0.4a28.tar.gz
xgboost-0.6a2.tar.gz
xgboost-0.4a26.tar.gz
xgboost-0.4a25.tar.gz
xgboost-0.4a30.tar.gz
xgboost-0.4a12.tar.gz
xgboost-0.4a14.tar.gz
xgboost-0.4a23.tar.gz
xgboost-0.4a19.tar.gz
xgboost-0.4a29.tar.gz
xgboost-0.6a1.tar.gz
xgboost-0.4a21.tar.gz
xgboost-0.4a13.tar.gz
xgboost-0.4a18.tar.gz
xgboost-0.4a24.tar.gz
xgboost-0.4a20.tar.gz
xgboost-0.4a27.tar.gz
```

I don't understand the logic in their versioning.